overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -10243,13 +10246,13 @@
           "lockedprogress:1": 1,
           "name:8": "§2Nitric Acid",
           "partysinglereward:1": 0,
-          "questlogic:8": "AND",
+          "questlogic:8": "OR",
           "repeat_relative:1": 1,
           "repeattime:3": -1,
           "simultaneous:1": 0,
           "snd_complete:8": "minecraft:entity.player.levelup",
           "snd_update:8": "minecraft:entity.player.levelup",
-          "tasklogic:8": "AND",
+          "tasklogic:8": "OR",
```

Comment:
Only one task, no need to make this `OR`

Reply:
Fixed, I think this was introduced because I initially didn't see the button for the requirement logic.